invenio_app_ils/ui/src/invenio_app_ils/common/components/DateRangePicker/DateRangePicker.js

```diff
@@ -0,0 +1,68 @@
+import React, { Component } from 'react';
+import { Header, Icon, Segment } from 'semantic-ui-react';
+import { DateInput } from 'semantic-ui-calendar-react';
+import PropTypes from 'prop-types';
+
+export class DateRangePicker extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      fromDate: props.defaultStart ? props.defaultStart : '',
+      toDate: props.defaultEnd ? props.defaultEnd : '',
+    };
+  }
+
+  handleDateChange = (event, { name, value }) => {
+    const fromDate = name === 'fromDate' ? value : this.state.fromDate;
+    const toDate = name === 'toDate' ? value : this.state.toDate;
+    this.setState({ [name]: value });
+    if (this.props.handleDateChange) {
+      this.props.handleDateChange(fromDate, toDate);
```

Comment:
I think it's good to include both `fromDate` and `toDate` but we won't know which one was changed. I think also passing `name` to `this.props.handleDateChange` would be useful.